server/src/test/java/io/crate/expression/scalar/formatting/ToCharFunctionTest.java

```diff
@@ -39,6 +39,20 @@ public void testEvaluateTimestamp() {
         );
     }
 
+    @Test
+    public void test_lower_case_patterns() throws Exception {
```

Comment:
Maybe worth adding test cases for all pattern which are not case-aware (=return same value independent of the case) and randomly use lower or upper case patterns? Since I already spotted typos this would be great and catch them.

Reply:
I added a test for all patterns that must behave in the same for lower/upper case in https://github.com/crate/crate/pull/17275/commits/b22879be33d44ee241c6f57f2ebf6490a7db5f40 and https://github.com/crate/crate/pull/17275/commits/95a319a5e25dba967a4928d039c5ffc6a34b871d